documentation/source/theory/rmg/dynamics.rst

```diff
@@ -0,0 +1,96 @@
+This allows it to pick up key low flux chain branching reactions and better represent
+species concentrations.
+
+Calculating the Dynamics Criterion
+==================================
+Let us define production and consumption rates for a given species
+:math:`\frac{dc_i}{dt} = P_i(t) - L_i(t)`
+
+Let us define a dimensionless concentration variable we will refer to as the
+accumulation number Ac for a given species
+
+:math:`Ac_{spc,i} = \frac{P_i}{L_i} \approx \frac{\bar{c_{i}}}{c_{i0}}`
+
+where :math:`\bar{c_i}` is the steady state concentration or more specifically the
+concentration at which :math:`P_i = L_i` assuming :math:`L_i` scales with :math:`c_i`
+and :math:`c_{i0}` is the current concentration.
+
+This species accumulation number is a measure of how far species i is from steady state.
+
+Since this number can only be calculated for core species, by itself it is only a
+measure of the behavior of species i within the reaction network.
+
+However if we consider models with and without some edge reaction j we can define
+
+:math:`\Pi_{Ac,i,j} = \frac{Ac_{spc,i,withj}}{Ac_{spc,i,withoutj}}`
+
+Which is a measure of how much the concentration of species i is impacted by
+reaction j.
+
+In order to directly compare multiple reactions we can then sum over all
+core species involved in reaction j to get our criterion the dynamics number.
+
+:math:`\sum_{i\in core} |Ln(\Pi_{Ac,i,j})| = Dy > \epsilon`
+
+Surface Algorithm
+=================
+One common issue with the dynamics criterion is that as can be seen in our theory
+section, it treats all core species equally.  Because of this, if the dynamics
+criterion is set too low it enters a feedback loop where it adds species and then
+since it can't get those species' concentrations right it adds more species and so on.
+In order to avoid this feedback loop the surface algorithm was developed.  It creates
+a new partition called the surface that is considered part of the core.  We will
+refer to the part of the core that is not part of the surface as the bulk core.  Operating
+without the dynamics criterion everything moves from edge to the bulk core as usual
+however the dynamics criterion is managed differently.  Lower dynamics criterion
+tolerances are set for movement from edge to surface than from edge to bulk core, with
+the constraint that edge reactions that don't either have all reactants in the bulk core
+or all products in the bulk core cannot enter the surface.  In general this prevents
+the dynamics criterion from adding reactions to get the concentrations of species
+(from the reactions) it pulled in correct avoiding the feedback loop.
+
+However, to avoid important species being trapped in the surface we also add criteria
+for movement from surface to bulk core based on flux or dynamics criterion.
+
+
+Key Parameters for Dynamics Criterion and Surface Algorithm
+===========================================================
+
+* toleranceMoveEdgeReactionToCore
```

Comment:
I would put the parameters in emphasis (double back quotes), use a colon after the parameter, and then delete the extra space between the description and the parameter. This will help emphasize when each new parameter starts.

Example (in rst)

\* \`\`toleranceMoveEdgeReactionToCore\`\`: An edge reaction will be pulled directly into the bulk core if its dynamics number ever exceeds this value.

Reply:
I tried this, but the bullets get all squished together and it isn't as easy to read as the format I copied from the pruning section.

Reply:
It is still hard to distinguish the parameters from the text beneath it. If nothing else, I would \*\***bold**\*\* the text for the parameter names.